### DISCUSSION TITLE
How to add "fromVersion" to PR title reliably

### DISCUSSION THREAD

**Discussion by valentijnscholten** [renovatebot/renovate#12090] [Request Help]
Posting here as the related existing issues are locked.

Similar to the creator of #1708, I would like to add to the PR the title the current version of the dependency that is being upgraded by the PR. There's benefit in immediately being able to see if it is a major/minor/patch update. Or even just a container image hash update.

I didn't get my info from #1708, but can't remember from where I got it. Our config is like this:

```
    "commitMessageExtra": "from {{currentVersion}} to {{#if isMajor}}v{{{newMajor}}}{{else}}{{#if isSingleVersion}}v{{{toVersion}}}{{else}}{{{newValue}}}{{/if}}{{/if}}",
    "commitMessageSuffix": "({{packageFile}})",
```
(public config: https://github.com/DefectDojo/django-DefectDojo/blob/aab58b08e7f309158373e6ab68ec0cf2a8bb5bf7/.github/renovate.json#L12-L13)

This seems work fine, except for docker images it seems. We run into #7443, where merged PRs remain on the dashboard as "Ignored" PRs. My suspicion is that it might be because the from and to versions are identical, or maybe it's just because docker image hashes need to be handled differently.

Any thoughts on a working config for both "normal/semver" dependencies and docker image hashes?

**Comment by viceice** [comment]
Checkout this

https://github.com/renovatebot/renovate/blob/f5db1cb56a4d737d5d5045849dbb88a1187e65c2/lib/datasource/docker/index.ts#L34-L59
